doc/ics/yaml/tbr_reutlingen_de.yaml

```diff
@@ -3,11 +3,18 @@ title: Technischer Betriebsdienst Reutlingen
 url: https://www.tbr-reutlingen.de
 howto:
   en: |
-    - Go to <https://www.tbr-reutlingen.de/entsorgungskalender> and select your street.  
-    - Right-click on `Abfuhrtermine (iCal) herunterladen` and copy link address.
+    - Open <https://www.tbr-reutlingen.de/entsorgungskalender> and go to section `Entsorgungskalender 2025`.
+    - Select your street, desired waste types and timeperiod.
+    - Right-click on `ICS` and copy link address.
     - Replace the `url` in the example configuration with this link.
```

Comment:
I don't like this line anymore and try to remove it every time I touch an ICS source. As it also shows up in the GUI configuration where there is no example. I'd prefer something like "Use this URL as the `url` parameter". (same for the German one)

Reply:
Changed the text as requested